cmake/apple/install_apple.sh.in

```diff
@@ -21,20 +21,16 @@ MSG_COLOR='\x1B[1;36m'
 COLOR_RESET='\x1B[0m'
 echo -e "$MSG_COLOR\n\nCreating App Bundle \"$APP\"...$COLOR_RESET"
 
-# Locate macdeployqt, assume homebrew & Qt5
-which macdeployqt > /dev/null 2>&1
```

Comment:
@zapashcanon as I mentioned in the comments, let's drop MacPorts support for now.  We'll keep these changes bu also make these changes please as they're MacPort-only.

```diff
- # STK rawwaves directory
- STK_RAWWAVE_LIST=( $HOME/stk-*/rawwaves )
- STK_RAWWAVE=${STK_RAWWAVE_LIST[0]}
-
- if [ ! -d "$STK_RAWWAVE" ]; then
     STK_RAWWAVE=$(brew --prefix stk)/share/stk/rawwaves
- fi
```